packages/ui-core/theme/src/lib/components/breadcrumb/breadcrumb.component.scss

```
@@ -0,0 +1,112 @@
+@use '@nebular/theme/styles/global/breakpoints' as *;
+@use 'themes' as *;
+
+// shadcn-style trail: small muted text, thin chevrons, no title styling and no
+// background block. Every value is a theme token so light and dark stay in sync.
+@include nb-install-component() {
+  display: flex;
+  align-items: center;
+  // Never push the header actions around — the trail shrinks and ellipsizes instead.
```

Stylelint (17.14.0)

[error] Unknown word Never (CssSyntaxError)

(CssSyntaxError)